src/oxrse_unit_conv/si.py

```diff
@@ -31,3 +31,7 @@
 
 candela = classes.SIUnit("candela", "cd")
 cd = candela
+
+Litre = classes.SIUnit("liter", "L")
```

Comment:
@Cetus137 Incorrect spelling according to the international bureau of measuring things